src/Schema/ArticleSchema.php

```diff
@@ -7,6 +7,9 @@
 use Illuminate\Support\HtmlString;
 use RalphJSmit\Laravel\SEO\Support\SEOData;
 
+/**
+ * @deprecated Use CustomSchema paradigm
+ */
```

Comment:
To be honest I don't think I want to deprecate this. I understand that CustomSchema gives you now the ability to do the same, but you lose IDE autocomplete compared to this and also now the developer is responsible for generating valid schema instead of the package. These two (or now three) are very common use cases so I would find it really convenient to have he auto-complete on it with fluent methods.